bin/trigger_azure_pipelines.py

```diff
@@ -68,6 +71,35 @@ def trigger_azure_pipeline_and_wait_until_its_completed(
     if completed_run.result in FAILED_STATUSES:
         raise Exception("Azure DevOps pipeline run failed!")
 
+    return run_id
+
+
+def download_callback(chunk, response) -> None:

```

Comment:
If you can find the types for these parameters it might be interesting to annotate them, but that's just a suggestion.